warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/SplitsFileType.java

```diff
@@ -0,0 +1,47 @@
+package datawave.ingest.mapreduce.job;
+
+/**
+ * Splits Files created will have the form:
+ * 
+ * <pre>
+ * tableA split1 [tablet]
+ * tableA split2 [tablet]
+ * ...
+ * tableB split1 [tablet]
+ * tableB split2 [tablet]
+ * ...
+ * </pre>
+ */
+public enum SplitsFileType {
+    // @formatter:off
+    /*
+     * Indicates that the splits are sampled to a constrained interval
+     * 
+     * For example if there are 10 splits: 
+     * tableA 0ab 
+     * tableA 1cd 
+     * tableA 2ef 
+     * tableA 3gh 
+     * tableA 4ij 
+     * tableA 5kl 
+     * tableA 6mn 
+     * tableA 7op 
+     * tableA 8qr
+     * tableA 9st
+     * 
+     * And the number wanted is 4 then the resulting splits would be 
```

Comment:
It seems that in practice there would be 5 total groups covered by these 4 splits.  Data for tablet 0ab would be included in the group ending with 2ef.  Tablet 9st would need to be included in an unnamed "leftover" group.

If the goal is to have n groups, it would make sense to have n-1 splits.  Otherwise, to have the number of groups match the number of splits, it might be cleaner and simpler to always include the final entry:
```
     * tableA 2ef 
     * tableA 4ij 
     * tableA 6mn 
     * tableA 9st
```

Reply:
I interpreted TRIMMEDBYNUMBER to mean down-sampled to a specified number of splits with it already implied that the number of groups would be n+1